### PR TITLE
Allow direct, internal access to elastic search

### PR DESCRIPTION
**Pull request by paulmr**
To allow for logfile processing (e.g. migration monitoring).

### CODE REVIEW

File: cloudformation/elk-stack.template [hunk 1]
```diff
@@ -439,6 +444,12 @@
                         "FromPort": "6379",
                         "ToPort": "6379",
                         "CidrIp": "0.0.0.0/0"
+                    },
+                    {
+                        "IpProtocol": "tcp",
+                        "FromPort": "80",
+                        "ToPort": "80",
+                        "CidrIp": "10.249.0.0/16"
```
Comment (mchv) [inline cloudformation/elk-stack.template]: What is `10.249.0.0/16` exactly? may it be configurable as a parameter of the script

File: cloudformation/elk-stack.template [hunk 2]
```diff
@@ -263,6 +263,11 @@
                         "Protocol": "TCP",
                         "LoadBalancerPort": "6379",
                         "InstancePort": "6379"
+                    },
+                    {
+                        "Protocol": "TCP",
+                        "LoadBalancerPort": "80",
+                        "InstancePort": "80"
```
Comment (mchv) [inline cloudformation/elk-stack.template:275]: May we create a separate security group for this rule as `Allow logstash messages to internal ELB` does not really match any more with the new rule?

Comment (paulmr) [inline cloudformation/elk-stack.template:275]: Personally, I like having all of the rules that describe the permissions for the internal ELB in one place ... but of course you are 100% correct that the description is now wrong. I will update it.

File: cloudformation/elk-stack.template (hunk 1 above)
Comment (paulmr) [inline cloudformation/elk-stack.template]: Good idea, I will make this a parameter.

**Comment by paulmr** [comment]
are the changes OK now @mchv ?

File: cloudformation/elk-stack.template (hunk 2 above)
Comment (mchv) [inline cloudformation/elk-stack.template:275]: @paulmr thanks for updating, fine with me!

**Comment by mchv** [comment]
@paulmr 👍

**Pull request merged**